readme.md

```diff
@@ -18,7 +18,8 @@ var BrowserStack = require( "browserstack" );
 // REST API
 var client = BrowserStack.createClient({
 	username: "foo",
-	password: "p455w0rd!!1"
+	password: "p455w0rd!!1",
+	proxy (optional): "http://myproxy.com:1234"
```

Comment:
I'd say the option should be documented, but should not be used in the sample code.

This is similar to the `host` option in my wordpress module. It's documented in [`createClient()`](https://github.com/scottgonzalez/node-wordpress#wordpresscreateclient-settings-), but not shown in the [usage example](https://github.com/scottgonzalez/node-wordpress#usage).